compiler/rustc_save_analysis/src/lib.rs

```diff
@@ -825,7 +825,7 @@ impl<'tcx> SaveContext<'tcx> {
         for attr in attrs {
             if let Some(val) = attr.doc_str() {
                 // FIXME: Should save-analysis beautify doc strings itself or leave it to users?
```

Comment:
Sounds like another place we can improve performance :)

Reply:
True. :)